src/html5shiv.js

```diff
@@ -30,7 +30,7 @@ define(['isSVG'], function(isSVG) {
       var expando = '_html5shiv';
 
       /** The id for the the documents expando */
-      var expandoID = 0;
+      var expanID = 0;
```

Comment:
this is not our code. do not change any of the file

Reply:
same as above